modules/las/docs/api-reference/las-loader.md

```diff
@@ -23,7 +23,8 @@ const data = await load(url, LASLoader, options);
 
 ## Options
 
-| Option               | Type     | Default | Description                                                               |
-| -------------------- | -------- | ------- | ------------------------------------------------------------------------- |
-| `options.las.skip`   | Number   | `1`     | Read one from every _n_ points.                                           |
-| `options.onProgress` | Function | -       | Callback when a new chunk of data is read. Only works on the main thread. |
+| Option                   | Type             | Default | Description                                                               |
+| ------------------------ | ---------------- | ------- | ------------------------------------------------------------------------- |
+| `options.las.skip`       | Number           | `1`     | Read one from every _n_ points.                                           |
+| `options.las.colorDepth` | Number or string | `16`    | Are colors encoded on 8 or 16 bits ? Can be set to `'auto'`.              |
```

Comment:
Setting the default to `16` (while correct) would be a breaking change. We should keep it at `8` for now to avoid surprising users, you can leave a comment in the code to fix for 3.0.

`Whether colors encoded using 8 or 16 bits? Can be set to `8`, `16`, or `'auto'`. Note: LAS recommends 16 bits.`